source/Pages/compute_on_spider.rst

```diff
@@ -221,17 +225,26 @@ Job types
   * For example, by specifying ``-c 2`` you request 2 cores and 160 GB scratch disk
   * When you target specifically our fat nodes with 12TB available scratch, the provided scratch disk per requested core is 200 GB
 
+GPU jobs
```

Comment:
Perhaps state here somewhere that the GPUs are only available for those user who have specifically requested and received access. In case access is required but not (yet) provided then user should contact the helpdesk for assistance